scripts/test262-runner.js

```diff
@@ -1088,6 +1129,82 @@ function runTest(
   }
 }
 
+function executeTestUsingSerializer(
+  test: TestFileInfo,
+  testFileContents: string,
+  data: BannerData,
+  // eslint-disable-next-line flowtype/no-weak-types
+  harnesses: Object,
+  strict: boolean,
+  timeout: number
+) {
+  let sources = [];
+
+  // Add the test262 intrinsics.
+  sources.push({
+    filePath: "test262.js",
+    fileContents: `\
+var $ = {
+  evalScript: () => {}, // noop for now
+  global,
+};
+var $262 = $;
+var print = () => {}; // noop for now
+  `,
+  });
+
+  // Add the harness files.
+  for (let name of ["sta.js", "assert.js"].concat(data.includes || [])) {
+    let harness = harnesses[name];
+    sources.push({ filePath: name, fileContents: harness });
+  }
+
+  // Add the test file.
+  sources.push({ filePath: test.location, fileContents: (strict ? '"use strict";' + EOL : "") + testFileContents });
+
+  let result;
+  try {
+    result = prepackSources(sources, {
+      serialize: true,
+      timeout: timeout * 1000,
+      errorHandler: diag => {
+        if (diag.severity === "Information") return "Recover";
+        if (diag.severity !== "Warning") return "Fail";
+        return "Recover";
+      },
+      onParse: () => {
+        // TODO(calebmer): Turn all literals into abstract values
```

Comment:
I guess behind some other option(s) that would allow to different levels of fuzzing?

Reply:
👍

Reply:
Yep